src/Nowruz/pages/disputes/disputeDetail/index.tsx

```diff
@@ -96,7 +96,16 @@ export const DisputeDetail = () => {
             {/* TODO: ADD anonymous chat function */}
             {dispute.direction === 'juror' && (
               <div className="flex flex-col gap-3 mt-8">
-                <Button variant="text" color="primary" customStyle="flex gap-2 !h-fit !p-0" fullWidth>
+                <Button
```

Comment:
you can use `Link` instead of Button cause styling is same and you can prevent to use `window.open` directly